### ISSUE TITLE
[AA4AM] Custom Connector - Get Organizations error out

### ISSUE THREAD

**Issue by rajyraman** [microsoft/coe-starter-kit#418]
After setting up a new custom connector, _GetOrganisations_ fails with "Resource not found", but _GetProjects_ works if I pass Organization parameter

![image](https://user-images.githubusercontent.com/5035266/119088309-d0016d00-ba4b-11eb-9331-d5f72cea875d.png)

**Comment by rajyraman** [comment]
It seems to be working now.

**Comment by Chris-WP** [comment]
@rajyraman I am experiencing exactly the same issue where GetProjects provides a success response, but GetOrganizations returns a "Resource not found" error.

Was there an extra step or workaround that you used to resolve the issue?

**Comment by rajyraman** [comment]
@Chris-WP - I had multiple accounts in the same browser session. So, I am not sure whether it had some part in this failing. I really don't know how it started working, as this was more than a month ago. Maybe try this in incognito first.

**Comment by Chris-WP** [comment]
The steps I performed to resolve the issue were:

1. Uninstalled "ALM Accelerator for Advanced Makers" solution
2. Deleted all connectors and connections relating to the "ALM Accelerator for Advanced Makers" solution
3. Reimported the "ALM Accelerator for Advanced Makers" solution
4. Opened the "CustomAzureDevOps" connection _from within the "ALM Accelerator for Advanced Makers" solution_
5. Updated the credentials for the "CustomAzureDevOps" connection
6. Opened the ALM Accelerator for Advanced Makers application
